### ISSUE TITLE
MetricsQL: `median_over_time` doesn't work in `aggr_over_time`

### ISSUE THREAD

**Issue by vpedosyuk** [VictoriaMetrics/VictoriaMetrics#5034]
### Describe the bug

I'm trying to run the following query:
```
aggr_over_time(("avg_over_time", "median_over_time"), node_arp_entries[1h])
```

But it throws the following error:
```
422 invalid args to aggr_over_time(("avg_over_time", "median_over_time"), node_arp_entries[1h]): "median_over_time" cannot be used in `aggr_over_time` function; expecting quoted aggregate function name
```

Since `median_over_time` is a rollup function I'd expect it to work fine with the `aggr_over_time`.

### To Reproduce

Here's a demo link:
https://play.victoriametrics.com/select/accounting/1/6a716b0f-38bc-4856-90ce-448fd713e3fe/prometheus/graph/?#/?g0.range_input=30m&g0.end_input=2023-09-21T08%3A30%3A38&g0.tab=0&g0.relative_time=last_30_minutes&g0.expr=aggr_over_time%28%28%22avg_over_time%22%2C+%22median_over_time%22%29%2C+node_arp_entries%5B1h%5D%29

### Version

Since it's reproduced on https://play.victoriametrics.com, I believe it's latest `v1.93.5`. But in our case we still have `v1.82.1` :melting_face:.

**Comment by valyala** [comment]
@vpedosyuk , thanks for filing the bug report! It should be fixed in the commit a7401595415b33e68eefd6594f598037e388b52f , which will be included in the next release. In the mean time you can build VictoriaMetrics from this commit according to [these instructions](https://docs.victoriametrics.com/#how-to-build-from-sources) and verify whether [median_over_time](https://docs.victoriametrics.com/MetricsQL.html#median_over_time) function can be passed to [aggr_over_time](https://docs.victoriametrics.com/MetricsQL.html#aggr_over_time) now.

If you run VictoriaMetrics cluster, then you can build cluster components (vminsert, vmselect and vmstorage) from the commit c954019e438515b47f1898cdfac82c61b1692336 according to [these docs](https://docs.victoriametrics.com/Cluster-VictoriaMetrics.html#building-from-sources).

**Comment by vpedosyuk** [comment]
@valyala thanks for the quick fix!

**Comment by valyala** [comment]
FYI, [median_over_time](https://docs.victoriametrics.com/MetricsQL.html#median_over_time) can be passed to [aggr_over_time](https://docs.victoriametrics.com/MetricsQL.html#aggr_over_time) starting from [v1.94.0](https://docs.victoriametrics.com/CHANGELOG.html).

**Comment by valyala** [comment]
FYI, the fix has been back-ported to [v1.93.6 LTS release](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.93.6).
